Specs/Number/Japanese/NumberModel.json

```diff
@@ -8583,6 +8571,7 @@
     "Input": "毎日この薬を〇・二五ミリリットル飲んでください。",
     "IgnoreResolution": true,
     "NotSupportedByDesign": "javascript, java, python",
+    "NotSupported": "dotnet",
```

Comment:
Why is this case now not supported? If this is a temporary side effect, please add a "Comment" attribute in the test spec describing the reason.

Reply:
Sure, we'll update that.